yogstation/code/game/objects/items/fishing/rods.dm

```diff
@@ -0,0 +1,149 @@
+/obj/item/twohanded/fishingrod
+	name = "fishing rod"
+	desc = "A fishing rod used for fishing. Despite looking ordinary, fishing has evolved for the cosmos with features like auto-reeling."
+	icon = 'yogstation/icons/obj/fishing/fishing.dmi'
+	icon_state = "fishing_rod"
+	lefthand_file = 'icons/mob/inhands/equipment/fishing_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	usesound = 'sound/items/crowbar.ogg'
+	slot_flags = ITEM_SLOT_BACK
+	force = 2
+	throwforce = 5
+	w_class = WEIGHT_CLASS_NORMAL
+	materials = list(/datum/material/iron=50)
+
+	var/fishing_power = 10
+	var/obj/item/reagent_containers/food/snacks/bait/bait = null //what bait is attached to the rod
+	var/fishing = FALSE
+	var/bobber_image = 'yogstation/icons/obj/fishing/fishing.dmi'
+	var/bobber_icon_state =  "bobber"
+	var/static/mutable_appearance/bobber = mutable_appearance('yogstation/icons/obj/fishing/fishing.dmi',"bobber")
+	var/datum/component/fishable/fishing_component
+	var/mob/fisher
+	var/bite = FALSE
+
+	
+
+/obj/item/twohanded/fishingrod/examine(mob/user)
+	. = ..()
+	. += "It's current fishing power is [fishing_power]."
+
+/obj/item/twohanded/fishingrod/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	var/datum/component/fishable/fc = target.GetComponent(/datum/component/fishable)
+	if(!fc)
+		return ..()
+	if(!fishing)
+		if(!wielded)
+			to_chat(user,"You need to wield the rod in both hands before you can cast it!")
+			return
+		cast(fc,user)
+	else
+		if(fc != fishing_component)
+			to_chat(user,"You are not fishing here!")
+			return
+		reel_in() //intentionally able to reel in with one hand
+
+/obj/item/twohanded/fishingrod/process()
+	if(!fishing)
+		return PROCESS_KILL
+	if(bite)
+		to_chat(fisher,"Whatever was on the line drifts back into the deep...")
+		bite = FALSE
+		return
+
+	if(prob(50))
+		to_chat(fisher,"Something bites! Reel it in!")
+		bite = TRUE
+		do_fishing_alert(fisher)
+
+/obj/item/twohanded/fishingrod/Destroy()
+	STOP_PROCESSING(SSobj,src)
+	return ..()
+
+/obj/item/twohanded/fishingrod/proc/cast(datum/component/fishable/fc,mob/user)
+	fishing = TRUE
+	fishing_component = fc
+	var/turf/fishing_turf = fishing_component.parent
+	fishing_turf.add_overlay(bobber)
+	fisher = user
+	RegisterSignal(user,COMSIG_MOVABLE_MOVED, .proc/reel_in_forced,TRUE)
+	RegisterSignal(src,COMSIG_MOVABLE_MOVED, .proc/reel_in_forced,TRUE)
+	RegisterSignal(src,COMSIG_ITEM_DROPPED, .proc/reel_in_forced,TRUE)
+	START_PROCESSING(SSobj,src)
+	playsound(fishing_component, 'sound/effects/splash.ogg', 50, FALSE, -5)
+	to_chat(fisher,"You cast out your fishing rod.")
```

Comment:
```suggestion
	to_chat(fisher, span_notice("<i>You cast out your fishing rod...</i>"))
```

Reply:
Find a span that does what you want, not using one that _almost_ does what you want and adding to it

Reply:
span_smalnoticeital("<FONT size = 2>"You cast out your fishing rod..."))

Reply:
then edit your suggestion

Reply:
this is better

Reply:
do you like smallnoticeital or notice(<i>